globals.go

```diff
@@ -538,6 +538,121 @@ const (
 	HostsNone          HostsPermission          = 0
 )
 
+func MapPermissionKeyToName(permissionKey string) string {
+	switch permissionKey {
+	case PermissionProjectSettings:
+		return "Settings"
+	case PermissionTasks:
+		return "Tasks"
+	case PermissionPatches:
+		return "Patches"
+	case PermissionLogs:
+		return "Logs"
+	case PermissionDistroSettings:
+		return "Distro Settings"
+	case PermissionHosts:
+		return "Distro Hosts"
+	}
+	return ""
+}
+
+type Permission struct {
+	Description string `json:"description"`
+	Value       int    `json:"value"`
+}
+
+func MapPermissionKeyToPermissionLevels(permissionKey string) []Permission {
```

Comment:
I wouldn't use the word Map here,

Reply:
also public functions should generally have a docs string

Reply:
changed it